scintilla/scintilla_changes.patch

```diff
@@ -84,6 +84,7 @@ index cd4b23617..af4a73db4 100644
 +		&lmCPP,
 +		&lmCss,
 +		&lmD,
++		&lmDart,
```

Comment:
As this isn't updating the chunk header, it's gonna corrupt the patch, isn't it?

Reply:
…well OK, it's already broken since at least a9363bc15d2c66ea91eeec923961f93e41bff216, but is that an excuse? :)

Reply:
Yeah, sorry, I've been blindly following the instructions in HACKING without thinking much for quite some time ;-).

I've just created https://github.com/geany/geany/pull/3991 to better describe how the patch file should be modified when adding a lexer.